### ISSUE TITLE
occ:repair doesn't work in NC 13.0.0*

### ISSUE THREAD

**Issue by enoch85** [nextcloud/server#7309]
### Steps to reproduce
1. Upgrade frm 12.0.3 --> 13.0.0beta1
2. Run occ:repair

### Expected behaviour
occ:repair should work

### Actual behaviour
```
An unhandled exception has been thrown:
Error: Call to a member function isEnabledForUser() on null in /var/www/nextcloud/core/Command/Maintenance/Repair.php:89
Stack trace:
#0 /var/www/nextcloud/3rdparty/symfony/console/Command/Command.php(264): OC\Core\Command\Maintenance\Repair->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#1 /var/www/nextcloud/3rdparty/symfony/console/Application.php(874): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 /var/www/nextcloud/3rdparty/symfony/console/Application.php(228): Symfony\Component\Console\Application->doRunCommand(Object(OC\Core\Command\Maintenance\Repair), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /var/www/nextcloud/3rdparty/symfony/console/Application.php(130): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 /var/www/nextcloud/lib/private/Console/Application.php(173): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#5 /var/www/nextcloud/console.php(96): OC\Console\Application->run()
#6 /var/www/nextcloud/occ(11): require_once('/var/www/nextcl...')
#7 {main}
Latest version is: 13.0.0beta1. Current version is: 13.0.0.
  - installed: true
  - version: 13.0.0.6
  - versionstring: 13.0.0 Beta 1
  - edition: 
```

## Server configuration detail

**Operating system:** Linux 4.4.0-101-generic #124-Ubuntu SMP Fri Nov 10 18:29:59 UTC 2017 x86_64

**Webserver:** Apache/2.4.18 (Ubuntu) (apache2handler)

**Database:** pgsql PostgreSQL 9.6.6 on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 5.4.0-6ubuntu1~16.04.4) 5.4.0 20160609, 64-bit

**PHP version:** 7.0.22-0ubuntu0.16.04.1
Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, session, standard, apache2handler, PDO, xml, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, iconv, imap, intl, json, ldap, exif, mcrypt, pdo_pgsql, pdo_sqlite, pgsql, Phar, posix, readline, redis, shmop, SimpleXML, smbclient, sockets, sqlite3, sysvmsg, sysvsem, sysvshm, tokenizer, wddx, xmlreader, xmlwriter, xsl, zip, libsmbclient, Zend OPcache

**Nextcloud version:** 13.0.0 Beta 1 - 13.0.0.6

**Updated from an older Nextcloud/ownCloud or fresh install:** 

**Where did you install Nextcloud from:** unknown

<details><summary>Signing status</summary>

Array
</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - activity: 2.6.1
 - admin_notifications: 1.0.1
 - calendar: 1.5.6
 - comments: 1.3.0
 - contacts: 2.0.1
 - dav: 1.4.5
 - federatedfilesharing: 1.3.1
 - federation: 1.3.0
 - files: 1.8.0
 - files_downloadactivity: 1.2.0
 - files_pdfviewer: 1.2.0
 - files_sharing: 1.5.0
 - files_texteditor: 2.5.1
 - files_trashbin: 1.3.0
 - files_versions: 1.6.0
 - files_videoplayer: 1.2.0
 - firstrunwizard: 2.2.1
 - gallery: 18.0.0
 - issuetemplate: 0.3.0
 - logreader: 2.0.0
 - lookup_server_connector: 1.1.0
 - mail: 0.7.5
 - nextcloud_announcements: 1.2.0
 - notes: 2.3.1
 - notifications: 2.1.0
 - oauth2: 1.1.0
 - password_policy: 1.3.0
 - previewgenerator: 1.0.7
 - provisioning_api: 1.3.0
 - qownnotesapi: 17.5.0
 - richdocuments: 1.12.37
 - serverinfo: 1.3.0
 - sharebymail: 1.3.0
 - survey_client: 1.1.0
 - systemtags: 1.3.0
 - theming: 1.4.1
 - twofactor_backupcodes: 1.2.3
 - updatenotification: 1.3.0
 - workflowengine: 1.3.0
Disabled:
 - admin_audit
 - encryption
 - files_external
 - user_external
 - user_ldap

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "updatechecker": false,
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "localhost",
        "192.168.4.111",
        "cloud.techandme.se",
        "office.techandme.se"
    ],
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "overwrite.cli.url": "https:\/\/cloud.techandme.se\/",
    "dbtype": "pgsql",
    "version": "13.0.0.6",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbtableprefix": "oc_",
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "logtimezone": "Europe\/Stockholm",
    "installed": true,
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpmode": "smtp",
    "mail_smtpauth": 1,
    "mail_smtpport": "587",
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpauthtype": "LOGIN",
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
    "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
    "allowZipDownload": true,
    "session_lifetime": 1800,
    "forcessl": true,
    "logfile": "\/var\/log\/nfs-share\/nextcloud.log",
    "loglevel": 2,
    "memcache.local": "\\OC\\Memcache\\Redis",
    "filelocking.enabled": true,
    "memcache.distributed": "\\OC\\Memcache\\Redis",
    "memcache.locking": "\\OC\\Memcache\\Redis",
    "redis": {
        "host": "***REMOVED SENSITIVE VALUE***",
        "port": 0,
        "timeout": 0,
        "dbindex": 0
    },
    "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
    "forwarded_for_headers": [
        "HTTP_X_FORWARDED",
        "HTTP_FORWARDED_FOR"
    ],
    "maintenance": false,
    "mail_smtpsecure": "tls",
    "htaccess.RewriteBase": "\/",
    "theme": "",
    "updater.release.channel": "beta",
    "trashbin_retention_obligation": "auto, 30",
    "activity_expire_days": 60
}
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

<details><summary>LDAP configuration (delete this part if not used)</summary>

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
</details>

## Client configuration

**Browser:** Mozilla/5.0 (X11; Linux x86_64; rv:57.0) Gecko/20100101 Firefox/57.0

**Operating system:** 

## Logs

<details><summary>Browser log</summary>

```
Insert your webserver log here 
```
</details>

<details><summary>Nextcloud log</summary>

```
Insert your Nextcloud log here

Error | PHP | Undefined variable: appManager at /var/www/nextcloud/core/Command/Maintenance/Repair.php#89 | 2017-11-27T16:57:53+0100
-- | -- | -- | --



```
</details>

<details><summary>Browser log</summary>

Insert your browser log here, this could for example include:

	a) The javascript console log
	b) The network log
	c) ...
</details>

**Comment by enoch85** [comment]
https://github.com/nextcloud/server/issues/7432 Related?

**Comment by tflidd** [comment]
Seems not to be related to pgsql: https://github.com/nextcloud/server/issues/7438

**Comment by enoch85** [comment]
Thanks for pointing that out @tflidd.

**Comment by enoch85** [comment]
Same in beta3.

**Comment by enoch85** [comment]
Please remove milestone if incorrect.

**Comment by rullzer** [comment]
PR for fix in #7560
